### PR TITLE
Group NO 8 

### PR DESCRIPTION
**Pull request by MoonRana**
What we Have done:
Mansoor Rehman      CIIT/SP21-BCS-042/ISB 
Meer Muneeb             CIIT/SP21-BCS-043/ISB 
Hannan Afzal              CIIT/SP21-BCS-026/ISB 
Mujtaba Tahir             CIIT/SP21-BCS-046/ISB

Teacher:
1:View Attempted Quiz
2:Add Quiz
3: Delate Quiz
4:Download Attempted Quiz

PHOTO-2023-12-24-18-41-13

### CODE REVIEW

**Comment by MoonRana** [comment]
Mansoor Rehman      CIIT/SP21-BCS-042/ISB 
Meer Muneeb             CIIT/SP21-BCS-043/ISB 
Hannan Afzal              CIIT/SP21-BCS-026/ISB 
Mujtaba Tahir             CIIT/SP21-BCS-046/ISB

File: models/quiz.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    type: Boolean,
+    default: false,
+  },
+});
+
+const Quiz = mongoose.model('Quiz', quizSchema);
+
+module.exports = Quiz;
+
+// FOR TESTING
```
Comment (amna-kamran) [inline models/quiz.js]: remove the commented code

File: routes/teacher.js [hunk 2]
```diff
@@ -1,4 +1,13 @@
-var express = require("express");
```
Comment (amna-kamran) [inline routes/teacher.js:1]: remove this line

File: routes/teacher.js [hunk 3]
```diff
@@ -1,4 +1,13 @@
...
-var router = express.Router();
```
Comment (amna-kamran) [inline routes/teacher.js:2]: remove this too

**Review by amna-kamran — APPROVED** [review]

File: routes/teacher.js (hunk 3 above)
Comment (MoonRana) [reply routes/teacher.js:2]: Done

**Pull request merged**